install.sh

```diff
@@ -127,7 +129,11 @@ function extract_install () {
 
     #extract and install
     echo "Extracting ${1} (this may take a while)..."
-    tar xpf ../"${2}"
+    if ! LD_LIBRARY_PATH=${CREW_PREFIX}/lib${LIB_SUFFIX}:/lib${LIB_SUFFIX} pixz -h > /dev/null; then 
+      tar xpf ../"${2}"
+    else
+    LD_LIBRARY_PATH=${CREW_PREFIX}/lib${LIB_SUFFIX}:/lib${LIB_SUFFIX} tar -Ipixz -xpf ../"${2}"
```

Comment:
Not sure why we would ever need to set LD_LIBRARY_PATH.  Are there system libraries needed for pixz and tar?